README.md

```diff
@@ -4,6 +4,12 @@ This is a maven multi-module project including all vaadin flow components.
 
 `master` branch is the latest version of all the components that will be released in the [Vaadin platform](https://github.com/vaadin/platform).
 
+## LICENSE
+SPDX-License-Identifier: Apache-2.0 OR Commercial Vaadin Add-on License version 3
+
+This project is dual-licensed under [Apache 2.0](http://www.apache.org/licenses/LICENSE-2.0) and [Commercial Vaadin Add-on License version 3](https://vaadin.com/license/cval-3.0) (CVALv3). 
```

Comment:
I would list the components under each license here somethning like ç

This project is dual-licensed, thus the license that applies depends on the component:
- [Apache 2.0](http://www.apache.org/licenses/LICENSE-2.0) for core components (vaadin-grid, vaadin ....)
- [Commercial Vaadin Add-on License version 3](https://vaadin.com/license/cval-3.0)  for pro components (vaadin-crud ... )

Reply:
the reason why i don't want to list all the components there is that I don't want to maintain the README file in the future, when we have modified one new/old component, also considering different branches have different component set. 

i think ``

Reply:
Done